mesheryctl/internal/cli/root/design/view.go

```diff
@@ -82,63 +82,142 @@ mesheryctl design view [design-name | ID]
 
 		req, err := utils.NewRequest("GET", url, nil)
 		if err != nil {
-			utils.Log.Error(err)
-			return nil
+			return err
 		}
 
 		res, err := utils.MakeRequest(req)
 		if err != nil {
-			utils.Log.Error(err)
-			return nil
+			return err
 		}
 
 		defer res.Body.Close()
 		body, err := io.ReadAll(res.Body)
 		if err != nil {
-			utils.Log.Error(utils.ErrReadResponseBody(err))
-			return nil
+			return utils.ErrReadResponseBody(err)
 		}
 
 		var dat map[string]interface{}
 		if err = json.Unmarshal(body, &dat); err != nil {
-			utils.Log.Error(utils.ErrUnmarshal(err))
-			return nil
+			return utils.ErrUnmarshal(err)
 		}
 
+		// Variables to track design structure for summary
+		var components []interface{}
+		var relationships []interface{}
+		designName := ""
+		sourceType := ""
+
 		if isID {
+			// Extract design info and find components/relationships
+			designName, sourceType = extractDesignInfo(dat)
+			components, relationships = findComponentsAndRelationships(dat)
+
+			hasComponents := len(components) > 0
+			hasRelationships := len(relationships) > 0
+
+			if !hasComponents {
+				utils.Log.Info("No components found for this design.")
+			}
+
+			if !hasRelationships {
+				utils.Log.Info("No relationships found for this design.")
+			}
+
 			if body, err = json.MarshalIndent(dat, "", "  "); err != nil {
-				utils.Log.Error(utils.ErrMarshalIndent(err))
-				return nil
+				return utils.ErrMarshalIndent(err)
 			}
 		} else if viewAllFlag {
 			// only keep the pattern key from the response when viewing all the patterns
 			if body, err = json.MarshalIndent(map[string]interface{}{"patterns": dat["patterns"]}, "", "  "); err != nil {
-				utils.Log.Error(utils.ErrMarshalIndent(err))
-				return nil
+				return utils.ErrMarshalIndent(err)
 			}
 		} else {
 			// use the first match from the result when searching by pattern name
-			arr := dat["patterns"].([]interface{})
-			if len(arr) == 0 {
-				utils.Log.Error(ErrDesignNotFound())
-				return nil
+			patternsVal, exists := dat["patterns"]
```

Comment:
Perhaps, we should avoid perpetuating use of the term "pattern", favoring "design" in its stead.